test/project_types/extension/commands/create_test.rb

```diff
@@ -69,7 +69,7 @@ def test_help_does_not_load_extension_project_type
         end
 
         output = io.join
-        refute_match('extension', output)
+        refute_match(' extension ', output)
```

Comment:
This may belong in a separate PR or require a different fix, but when the create command is hidden (see this [PR](https://github.com/Shopify/shopify-app-cli/pull/773)) this test passes as it was. When the create command is visible (after running `shopify config feature script_project --enable`) this test failed because the help for `shopify create script` has `--extension_point=TYPE` in it. I think this is a reasonable fix because the test makes sure that the extension project type isn't visible, which would mean that it shouldn't appear in `[ node | rails | script ]` and wouldn't have a space on either side.

Reply:
I think the better solution would be to check that `extension` is not a substring.  Maybe:
`refute(output.include?('extension'))`.

Reply:
I may have misunderstood, but I think this has the same issue as the original test because create script's help has the substring `extension` in it. Would `refute(output.include?(' extension '))` work or is that the same issue that `refute_match(' extension ', output)` had?